src/containers/Transactions/test/actions.test.js

```diff
@@ -58,26 +66,44 @@ describe('Transaction actions', () => {
         error: 'get_transaction_failed',
       },
     ];
-    store.dispatch(actions.loadTransaction(OfferCreateData.result.hash));
-    moxios.wait(() => {
-      const request = moxios.requests.mostRecent();
-      request
-        .respondWith({
-          status: 500,
-          response: {},
-        })
-        .then(() => {
-          expect(store.getActions()).toEqual(expectedActions);
-          done();
-        });
-    });
+    // this method sends a request to the server
+    store.dispatch(actions.loadTransaction(OfferCreateData.result.hash, client));
+    // console.log(server);
+
+    // when this is uncommented, it never resolves
+    // await server.connected;
```

Comment:
I think you probably still need this, but you need it before trying to take any actions.

Reply:
It hangs wherever it is